sample-multi-module/team-props/tasks.gradle

```diff
@@ -0,0 +1,13 @@
+task clean(type: Delete, dependsOn: subprojects.collect { p -> "$p.name:clean" }) {
+    delete project.buildDir
+}
+
+task check(dependsOn: subprojects.collect { p ->
+    "$p.name:check"
+})
+
+task assemble(dependsOn: subprojects.collect { p ->
+    "$p.name:assemble"
+})
+
+task build(dependsOn: ['assemble', 'check'])
```

Comment:
Are we sure we need these three tasks? Isn't it the default behaviour?

Reply:
right. the way we refer to the tasks just needs some changing.